nucypher/cli/types.py

```diff
@@ -37,7 +42,7 @@ def convert(self, value, param, ctx):
         try:
             _address = ip_address(value)
         except ValueError as e:
-            self.fail(str(e))
+            self.fail("Invalid IP Address")
```

Comment:
Nice to know what the problem was:

```suggestion
            self.fail(f"Invalid IP Address: {str(e)}")
```

Reply:
Meh - I think I prefer to just fail at this eager level.